templates/default/rt/rtticketeventlist.html

```diff
@@ -25,85 +25,21 @@
     <TBODY>
     <TR id="eventpanel-{$layout.module}" style="display: none;">
         <TD colspan="2">
-            <table class="lmsbox-inner">
+            <table class="lmsbox lms-ui-background-cycle event-day-table">
```

Comment:
To będzie miało obsługiwać przeciąganie zdarzeń? Jeśli nie to na wszelki wypadek wywal event-day-table.

Reply:
Usunięte.